install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -54,6 +54,9 @@
 {{- $bpfCtTcpMax := (coalesce .Values.bpf.ctTcpMax $defaultBpfCtTcpMax) -}}
 {{- $bpfCtAnyMax := (coalesce .Values.bpf.ctAnyMax $defaultBpfCtAnyMax) -}}
 {{- $kubeProxyReplacement := (coalesce .Values.kubeProxyReplacement $defaultKubeProxyReplacement) -}}
+{{- if and (ne $kubeProxyReplacement "disabled") (ne $kubeProxyReplacement "partial") (ne $kubeProxyReplacement "strict") }}
```

Comment:
any reason to have this check here instead of validation.yaml? I assume the $defaultKubeProxyReplacement is always having valid value.

Reply:
`$defaultKubeProxyReplacement` is computed only in this file, so I'm assuming it is not having the same values in `validation.yaml`, or at least that seems to be the case.